reviews/NFTTrader/Reentrancy.md

```diff
@@ -0,0 +1,71 @@
+# Reentrancy
+
+## By: Arcturus
+
+## Summary
+
+- The contracts use onlyOwner modifiers throughout to control access except for select unknown circumstances.
+
+### Recommendations
+
+-
+
+## Scope
+
+```
+├── Docs.docx
+└── test
+  └── 001_deployment_test.js
+└── contracts
+  ├── AXBridge.sol
+  ├── BatchSwap.sol
+  ├── BatchVault.sol
+  ├── CKBridge.sol
+  ├── ERC1155Five.sol
+  ├── ERC1155Four.sol
+  ├── ERC1155One.sol
+  ├── ERC1155Six.sol
+  ├── ERC1155Three.sol
+  ├── ERC1155Two.sol
+  ├── ERC20Four.sol
+  ├── ERC20One.sol
+  ├── ERC20Three.sol
+  ├── ERC20Two.sol
+  ├── ERC721Four.sol
+  ├── ERC721One.sol
+  ├── ERC721Three.sol
+  ├── ERC721Two.sol
+  ├── Pausable.sol
+  └── TradeSquads.sol
+```
+
+## File Review
+
+### `contracts/`
+
+#### `AXBridge.sol`
+
+- AXBridge is defined as Ownable with an onlyOwner modifier on its only function `bridgeSafeTransferFrom`. Although `bridgeSafeTransferFrom` makes an external function call `transferFrom`, on `bridgeSafeTransferFrom` the onlyOwner modifier appears to prevent it from being susceptible to reentrancy.
```

Comment:
appreciate it GrandGarcon I think a nonReentrant would be a good idea since like you mentioned it's unknown what these customInterfaces can actually do based on the dApp provided.